packages/mobile/src/localization/languages/ua/screens.ts

```diff
@@ -78,4 +78,31 @@ export const screens = {
     LAST_NAME_INVALID: 'Неправильний формат прізвища',
     WRONG_FORMAT: 'Невірний формат',
   },
+  make_a_post: {
+    TITLE: 'Створити публікацію',
+    DOWNLOAD_PHOTOS: 'Завантажити зображення',
+    ADD_PHOTOS: 'Додати фото',
+    DESCRIPTION: 'Опис',
+    DESCRIPTION_PLACEHOLDER: 'Ввeдіть опис товару',
+    CATEGORY: 'Категорія',
+    CATEGORY_PLACEHOLDER: 'Оберіть категорію',
+    TITLE_NAME: 'Назва товару',
+    TITLE_NAME_PLACEHOLDER: 'Введіть назву',
+    PRICE: 'Ціна',
+    PRICE_PLACEHOLDER: 'Введіть ціну',
+    PRICE_TOOLTIP: 'Інформація про ціну',
+    CURRENCY: 'Валюта',
+    CONTACT: 'Контактна інформація',
+    COUNTRY: 'Країна',
+    CITY: 'Місто',
+    MOBILE_PHONE: 'Телефон',
+    MOBILE_PHONE_PLACEHOLDER: '+380 ()',
```

Comment:
This isn't seemed to require translation)

Reply:
we could have different phone codes for different countries